support-frontend/assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -215,10 +206,11 @@ function ContributionFormContainer(props: PropTypes) {
         </div>
 
         <div className="gu-content__form">
-          {countryGroupDetails.tickerJsonUrl ?
+          {showTicker && campaign && campaign.tickerJsonUrl ?
```

Comment:
I feel like if a campaign is set up with a ticker, you shouldn't need to pass `ticker=true` in the URL as well - that will probably trip people up. That URL param is leftover from a time when we thought we might get away with configuring everything for the campaign entirely in the URL, which proved unfeasible.

I'd probably ditch `showTicker`, unless you want a way to force a ticker to display via the URL regardless of campaigns etc. In which case it should be an OR condition and you would need to supply the ticker type & json source URL params too, or default them

Reply:
👍